components/style/values/specified/position.rs

```diff
@@ -608,6 +610,37 @@ impl Parse for TemplateAreas {
 
 trivial_to_computed_value!(TemplateAreas);
 
+#[derive(Clone, Debug, PartialEq, ToCss)]
+/// Arc type for `Arc<TemplateAreas>`
+pub struct TemplateAreasArc(pub Arc<TemplateAreas>);
+
+impl Parse for TemplateAreasArc {
```

Comment:
We should be able to remove `Parse for TemplateAreas` now, right? Alternatively this method could be:

```rust
TemplateAreasArc(Arc::new(TemplateAreas::parse(context, input)?))
```

Reply:
I remember I `impl` this trait due to getting errors about lack of this trait.
I'll try to remove it to see if it's necessary.
Or, I'll try to update the method as returning a `TemplateAreasArc`.

Reply:
Hmm... yeah, I'll get error messages like following

```
error[E0599]: no function or associated item named `parse` found for type `values::Either<values::specified::position::TemplateAreasArc, values::None_>` in the current scope
     --> /servo/target/geckolib/debug/build/style-abfaa561961fdc53/out/properties.rs:40333:13
      |
40333 |             specified::GridTemplateAreas::parse(context, input)
      |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      |
      = note: the method `parse` exists but the following trait bounds were not satisfied:
              `values::specified::position::TemplateAreasArc : parser::Parse`
      = help: items from traits can only be used if the trait is implemented and in scope
      = note: the following traits define an item `parse`, perhaps you need to implement one of them:
              candidate #1: `parser::Parse`
              candidate #2: `bitflags::<unnamed>::str::StrExt`
              candidate #3: `style_traits::Separator`

error: aborting due to previous error
```

Let me try to update the method body like what you wrote.